src/chainparams.cpp

```diff
@@ -291,7 +291,7 @@ class CMainParams : public CChainParams {
         consensus.retiredBurnAddress = GetScriptForDestination(DecodeDestination("8defichainDSTBurnAddressXXXXaCAuTq", *this));
 
         // Destination for unused emission
-        consensus.unusedEmission = GetScriptForDestination(DecodeDestination("8Hw9iMi5u4qhfdb4ziod8gf7hHeMaexz8T", *this));
+        consensus.unusedEmission = GetScriptForDestination(DecodeDestination("df1qlwvtdrh4a4zln3k56rqnx8chu8t0sqx36syaea", *this));
```

Comment:
@prasannavl why this address? I would have expected it to be the address of the burn bot. but this is a new one?